Katello/katello
# Fixes #27772 - Use sync_history for content details on smart proxy

This PR removes pulp2 calls to display content details for a smart proxy and uses smart_proxy_history instead.

To test:

Create a few repos, cv etc and sync a smart proxy and go to content tab. The page should look like it does today on master.
